templates/tableau-single-server-ubuntu.template

```diff
@@ -293,7 +293,7 @@
         },
         "DefaultConfiguration": {
             "InstallationConfig": {
-                "TableauServerInstaller": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server_amd64.deb",
+                "TableauServerInstaller": "https://s3-us-west-2.amazonaws.com/tableau-quickstart/tableau-server-2018-1-1_amd64.deb",
```

Comment:
We should not be changing the installer name; otherwise we will need to update the templates every quarter.  I usually just drop the new installers and rename them to what the templates expect.